### DISCUSSION TITLE
开发者例会议题收集（20220830）

### DISCUSSION THREAD

**Discussion by chuntaojun** [polarismesh/polaris#606] [Meeting]
# 开发者例会议题收集（20220830）

## 会议信息

andrewshan(单家骏) 邀请您参加腾讯会议
会议主题：Polaris社区开发者双周会
会议时间：2022/09/06 19:30-20:00 (GMT+08:00) 中国标准时间 - 北京
重复周期：2022/07/12-2022/10/04 19:30-20:00, 每两周 (周二)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/dm/kBemx9IzxmGE

#腾讯会议：701-1815-5065

复制该信息，打开手机腾讯会议即可参与

## 议题列表

- https://github.com/polarismesh/polaris/issues/614
- https://github.com/polarismesh/polaris/issues/625
  - sig 中的 spec：https://github.com/nextarch/SIG-Microservice/blob/main/specification/v1/traffic-control-spec/router-zh.md

**Comment by andrewshan** [comment]
- https://github.com/polarismesh/polaris/issues/569
- https://github.com/polarismesh/polaris/issues/631

**Comment by Guo-Zhang** [comment]
Mock服务方案：https://github.com/polarismesh/polaris/issues/638

如果需要，我可以在会议上演示我这边完整的内部实践，不太方便直接写issue。
